src/patternfly/demos/Toolbar/examples/Toolbar.md

```diff
@@ -302,8 +302,8 @@ import './Toolbar.css'
     {{/toolbar-content-section}}
     {{> toolbar-expandable-content}}
   {{/toolbar-content}}
-  {{#> toolbar-content toolbar-content--modifier="pf-m-chip-container"}}
-    {{#> toolbar-item toolbar-item--modifier=(concat 'pf-m-chip-group ' toolbar-item-chip-group--modifier)}}
+  {{#> toolbar-content toolbar-content--modifier="pf-m-label-container"}}
+    {{#> toolbar-item toolbar-item--modifier=(concat 'pf-m-label-group ' toolbar-item-label-group--modifier)}}
```

Comment:
Similar to above, we don't have styles for these combinations of selectors, do we want to keep them on?